sklearn/metrics/ranking.py

```diff
@@ -259,7 +265,22 @@ def roc_auc_score(y_true, y_score, average="macro", sample_weight=None,
 
     max_fpr : float > 0 and <= 1, optional
         If not ``None``, the standardized partial AUC [3]_ over the range
-        [0, max_fpr] is returned.
+        [0, max_fpr] is returned. For the multiclass case, ``max_fpr``,
+        should be either equal to ``None`` or ``1.0`` as AUC ROC partial
+        computation currently is not supported for multiclass.
+
+    multiclass : string, 'ovr' or 'ovo', optional(default='ovr')
+        Determines the type of multiclass configuration to use.
+        ``'ovr'``:
+            Calculate metrics for the multiclass case using the one-vs-rest
+            approach.
+        ``'ovo'``:
+            Calculate metrics for the multiclass case using the one-vs-one
+            approach.
+
+    labels : array, shape = [n_classes] or None, optional (default=None)
```

Comment:
This is confusing. If this is always length ``n_classes`` why do we have this?

Reply:
This is covering the case where the cardinality of `y_true` is not the same as the the number of columns in `y_score`. This can happen when the `y_train` contains a class that is not in `y_test`.